pkg/services/libraryelements/database.go

```diff
@@ -148,14 +148,20 @@ func (l *LibraryElementService) createLibraryElement(c context.Context, signedIn
 	}
 
 	metrics.MFolderIDsServiceCount.WithLabelValues(metrics.LibraryElements).Inc()
+	// folderUID *string will be changed to string
+	var folderUID string
+	if cmd.FolderUID == nil {
+		folderUID = ""
```

Comment:
otherwise the `element.FolderUID` ends up always empty (the reason the `TestPatchLibraryElement` fails)
```suggestion
	if cmd.FolderUID != nil {
		folderUID = *cmd.FolderUID
```